metis-schema/src/main/java/eu/europeana/metis/schema/model/MediaType.java

```diff
@@ -12,50 +13,67 @@
   /**
    * Audio
    **/
-  AUDIO,
+  AUDIO("AUDIO"),
 
   /**
    * Video
    **/
-  VIDEO,
+  VIDEO("VIDEO"),
 
   /**
    * Text (including PDFs)
    **/
-  TEXT,
+  TEXT("TEXT"),
 
   /**
    * Graphical
    **/
-  IMAGE,
+  IMAGE("IMAGE"),
 
   /**
-   * Media that is not of any of the other kinds.
+   * 3D
+   */
+  THREE_D("3D"),
+
+  /**
+   * Media that is not any of the other kinds.
    **/
-  OTHER;
+  OTHER("OTHER");
+
 
   private static final String[] SUPPORTED_APPLICATION_TYPES_AS_TEXT = new String[]{
       "application/xml", "application/rtf", "application/epub", "application/pdf",
       "application/xhtml+xml"};
 
+  private final String stringRepresentation;
+
+  MediaType(String stringRepresentation) {
+    this.stringRepresentation = stringRepresentation;
+  }
+
+  @Override
+  @JsonValue
+  public String toString() {
+    return stringRepresentation;
+  }
+
   /**
-   * Obtains the media type of a given mime type. This method accepts media types with subtypes
-   * and/or parameters.
+   * Obtains the media type of a given mime type. This method accepts mime types with subtypes and/or parameters.
    *
-   * @param mediaType The media type.
+   * @param mimeType The media type.
    * @return The media type to which the mime type belongs.
    */
-  public static MediaType getMediaType(String mediaType) {
+  public static MediaType getMediaType(String mimeType) {
```

Comment:
Just a small detail but why change from `mediaType` to `mimeType`? For me makes makes more sense to use `mediaType`

Reply:
I'll disagree.
We call it MediaType as being a media of a type group we have defined.
That media type group is selected based on the mime type, and the mime type comes from the http protocol in fact and this is what is supposed to be passed in this method, so that we can classify it.

See https://developer.mozilla.org/en-US/docs/Web/HTTP/Basics_of_HTTP/MIME_types